ksp_plugin/manœuvre_body.hpp

```diff
@@ -43,6 +43,12 @@ Instant Manœuvre<InertialFrame, Frame>::final_time() const {
 }
 
 template<typename InertialFrame, typename Frame>
+bool Manœuvre<InertialFrame, Frame>::FitsBetween(Instant const& begin,
+                                                 Instant const& end) {
+  return begin < initial_time() && final_time() < end;
```

Comment:
Should this be semi-open (ie begin is in the interval and end is not)?  That would be more inline with the customary convention.